syntax = "proto3";

package producer;

service Producer
{
    rpc ProduceMsg (ProduceMsgRequest) returns (ProduceMsgReply) {}
}

message ProduceMsgRequest
{
    uint32 msg_id = 1;
    uint64 start_uid = 2;
    uint64 end_uid = 3;
}

message ProduceMsgReply
{
    int32 err = 1;
}

message ProducerMsg
{
    uint64 to_uid = 1;
    uint32 msg_id = 2;
}
